sopel/config/core_section.py

```diff
@@ -712,24 +991,51 @@ def homedir(self):
     """The number of seconds acceptable between pings before timing out.
 
     :default: ``120``
+
+    You can increase the timout (for example up to 200s) like this:
+
+    .. code-block:: ini
+
+        timeout = 200
+
     """
 
     use_ssl = ValidatedAttribute('use_ssl', bool, default=False)
     """Whether to use a SSL/TLS encrypted connection.
 
     :default: ``False``
+
+    Example with SSL on:
+
+    .. code-block:: ini
+
+        use_ssl = yes
+
     """
 
     user = ValidatedAttribute('user', default='sopel')
     """The "user" for your bot (the part before the ``@`` in the hostname).
 
     :default: ``sopel``
 
-    **Required.**
+    **Required**:
+
+    .. code-block:: ini
+
+        user = sopel
+
     """
 
     verify_ssl = ValidatedAttribute('verify_ssl', bool, default=True)
     """Whether to require a trusted certificate for encrypted connections.
 
     :default: ``True``
+
+    Example with SSL on:
+
+    .. code-block:: ini
+
+        use_ssl = yes
+        verify_ssl = yes
```

Comment:
```suggestion
        use_ssl = True
        verify_ssl = True
```

Same editorial comment as `log_raw`.